pkg/kubelet/container/runtime_test.go

```diff
@@ -0,0 +1,151 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package container
+
+import (
+	"testing"
+)
+
+func TestParseValidContainerID(t *testing.T) {
+	cID := ParseContainerID("test://some-container-id")
```

Comment:
thanks very much for your effort :). Please use `for loops` for tests, refer to https://github.com/kubernetes/kubernetes/pull/39568/files#diff-5a979eff174acb57bf1b20dc522651e2R3491 for detail.